docs/source/tools/acquire.rst

```diff
@@ -5,56 +5,64 @@ acquire
 This makes ``acquire`` an excellent tool to, among others, speed up the process of digital forensic triage.
 ``acquire`` uses ``dissect`` to gather forensic artifacts from the raw disk, if possible.
 
-``acquire`` gathers artifacts based on modules. These modules are paths or globs on a filesystem which acquire attempts to gather.
-Multiple modules can be executed at once, which have been collected together inside a profile.
-These profiles (used with ``--profile``) are  ``full``, ``default``, ``minimal`` and ``none``.
-Depending on what operating system gets detected, different artifacts are collected.
-
 The most basic usage of ``acquire`` is as follows:
 
 .. code-block:: console
 
     $ sudo acquire
 
-The tool requires administrative access to read raw disk data instead of using the operating system for file access.
-However, there are some options available to use the operating system as a fallback option. (e.g ``--fallback`` or ``--force-fallback``)
+The tool requires administrative access to read raw disk data.
+However, there are some options available to use the local operating systems's filesystem as a fallback option. (e.g ``--fallback`` or ``--force-fallback``)
+
+
+Filesystem acquisition
+----------------------
+
+``acquire`` gathers artifacts based on modules.
+These modules contain paths or globs that ``acquire`` attempts to gather from a filesystem.
+``acquire`` can execute multiple modules at once, where a collection of them are combined into a profile.
```

Comment:
```suggestion
``acquire`` can execute multiple modules at once (comma separated). Instead of specifying modules manually, it's possible to use a predefined collection known as a profile.
```

Reply:
oh yes, my mistake